### PR TITLE
Added Generic OnOff messages with tests

### PR DESCRIPTION
**Pull request by rafalgajda-silvair**

### CODE REVIEW

File: bluetooth_mesh/messages/generic/generic_onoff_message.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
+from construct import Int8ul, Struct, Switch, Optional, this
+from enum import IntEnum
+from bluetooth_mesh.messages.util import EnumAdapter, Opcode
+from bluetooth_mesh.messages.generics import Delay, TransitionTime
+
+
+class GenericOpcode(IntEnum):
```
Comment (michallowasrzechonek-silvair) [inline bluetooth_mesh/messages/generic/generic_onoff_message.py]: Rename to `GenericOnOffOpcode`

File: bluetooth_mesh/messages/generic/generic_onoff_message.py [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    ONOFF_GET = 0x8201
+    ONOFF_SET = 0x8202
+    ONOFF_SET_UNACKNOWLEDGED = 0x8203
+    ONOFF_STATUS = 0x8204
+
+
+GenericOnOffMsg = Struct(
```
Comment (michallowasrzechonek-silvair) [inline bluetooth_mesh/messages/generic/generic_onoff_message.py]: rename to `GenericOnOffSet`

File: bluetooth_mesh/messages/generic/generic_onoff_message.py [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    "remaining_time" / Optional(TransitionTime)
+)
+
+
+GenericOnOffMessage = Struct(
+    "opcode" / EnumAdapter(Opcode, GenericOpcode),
+    "params" / Switch(
+        this.opcode,
+        {
+            GenericOpcode.ONOFF_GET: Struct(),
```
Comment (michallowasrzechonek-silvair) [inline bluetooth_mesh/messages/generic/generic_onoff_message.py]: Extract as `GenericOnOffGet`

File: bluetooth_mesh/messages/test/test_generics.py [hunk 4]
```diff
@@ -1,6 +1,7 @@
 import pytest
 from construct import ValidationError
 
+from bluetooth_mesh.messages.generic.generic_onoff_message import *
```
Comment (michallowasrzechonek-silvair) [inline bluetooth_mesh/messages/test/test_generics.py]: `bluetooth_mesh.messages.generic.onoff`

File: bluetooth_mesh/messages/generic/generic_onoff_message.py (hunk 1 above)
Comment (rafalgajda-silvair) [reply bluetooth_mesh/messages/generic/generic_onoff_message.py]: Done.

File: bluetooth_mesh/messages/generic/generic_onoff_message.py (hunk 2 above)
Comment (rafalgajda-silvair) [reply bluetooth_mesh/messages/generic/generic_onoff_message.py]: Done.

File: bluetooth_mesh/messages/generic/generic_onoff_message.py (hunk 3 above)
Comment (rafalgajda-silvair) [reply bluetooth_mesh/messages/generic/generic_onoff_message.py]: Done.

File: bluetooth_mesh/messages/test/test_generics.py (hunk 4 above)
Comment (rafalgajda-silvair) [reply bluetooth_mesh/messages/test/test_generics.py]: Done.

**Pull request merged**